src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.dao;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionDao {
+    MovieSession add(MovieSession movieSession);
+
+    Optional<MovieSession> get(Long id);
+
+    List<MovieSession> findMovieSessionsByMovieIdAndTime(Long movieId, LocalDateTime timeFrom,
+                                                         LocalDateTime timeTo);
```

Comment:
```suggestion
    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
```
you can use SQL date() function in the query